src/pages/certificates/BrowserImport.tsx

```diff
@@ -132,6 +152,33 @@ const BrowserImport: FC = () => {
             </ul>
           </div>
         )}
+
+        {activeTab === MACOS && (
+          <div role="tabpanel" aria-label="safari macos">
```

Comment:
Is this guide for Safari only, or for any macOS browser? It's currently quite unclear what browser(s) this tab is applicable to, and which operating systems the other tabs are applicable to - except the 2 ones that state it explicitly in parentheses.
For example, if I'm using Firefox on macOS, which tab should I select? The "Firefox" one or  the "macOS" one? This is not clear to me at the moment. I thought we decided to have a double selector: a first one for the OS and a second one for the browser. Did I misunderstand anything?

Reply:
We didn't find a good solution. With two dropdowns, you would need just so many clicks to get to the right browser/os combination. Mixing tabs and a dropdown is a very uncommon pattern.

But I see your point that we are not super clear here. Then again, on macOS there is no cert import in firefox, so I guess users will find the right section even with this ambiguity?

Reply:
I hope so. It's not too important to find a solution now. We can improve this in a future iteration, hopefully after receiving some other good feedback like the one that prompted these changes. :slightly_smiling_face: